intellisense.md

```diff
@@ -63,6 +57,13 @@ This also works for variables defined in the file:
 
 ![](media/intellisense_variable_completion.png)
 
-It's important to note that today, we only support IntelliSense for functions
-_in the same file_. In the future, we will be adding support for IntelliSense
-across your entire project.
+IntelliSense behaves subtly differently depending on where you are currently typing:
+
+* **IntelliSense and the Interactive Window**: When typing in the Interactive
+Window, IntelliSense will only consider names that are defined in your current R
+session. IntelliSense will not look in the files within your project for names
+that are defined there.
+
+* **IntelliSense and the Editor**: When typing in an editor window, IntelliSense
+will look at **both** the contents of the current file, and any names defined
+in your current R session.
```

Comment:
Also, Survey/News is still 

![image](https://cloud.githubusercontent.com/assets/12820357/23911833/3a2b7462-089b-11e7-93b2-d5de15fecd43.png)